core/templates/dev/head/services/stateful/FocusManagerServiceSpec.js

```diff
@@ -71,7 +71,8 @@ describe('Focus Manager Service', function() {
     FocusManagerService.setFocusIfOnDesktop(focusLabel);
     if (!DeviceInfoService.isMobileDevice()) {
       $timeout(function () {
-        expect(rootScope.$broadcast).toHaveBeenCalledWith('focusOn',focusLabel);
+        expect(rootScope.$broadcast).toHaveBeenCalledWith('focusOn',
```

Comment:
If contents of (...) span multiple lines, break after the "(" char. (Would be nice if there is a lint time to enforce this.)

Reply:
Okay, got it! I'll look into a check for this as well. Thanks!